README.md

```diff
@@ -1,33 +1,46 @@
-# july-challenge-tbd
+# Getting Started with Create React App
 
-A web app that acts as a directory to find employees based on their areas of expertise and team involvement.
```

Comment:
I'd also keep this line included so we have a brief description to expand upon later!